layers/eight_mile/calibration/calibration.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+from collections import namedtuple
+import numpy as np
+
+__all__ = ["calibration_bins", "multiclass_calibration_bins", "binary_calibration_bins", "average_confidence", "Bins"]
+
+
+Bins = namedtuple("Bins", "accs confs counts edges")
+
+
+def multiclass_calibration_bins(true: np.ndarray, probs: np.ndarray, bins: int, class_weights: Optional[np.ndarray] = None) -> Bins:
+    """Calculate the binned confidence and accuracy for a multiclass problem.
+
+    :param true: A 1D array of the true labels for some examples.
+    :param probs: A 1D array of the probabilities from a model. Each row represents an example in the dataset.
+        and each column represents the probably assigned by the model to each class for that example.
+    :param bins: The number of bins to use when aggregating.
+    :param class_weights: A 1D array of scores that can add extra weight to examples of specific classes.
+
+    :returns: The metrics aggregated by bins
+    """
+    preds = np.argmax(probs, axis=1)
+    pred_probs = np.max(probs, axis=1)
+    correct = true == preds
```

Comment:
I think we should rename this param.

Reply:
Any idea to what? I already changed to to correct like you mentioned before, I guess it is a little odd now that it can have a score on it

Reply:
I mean `true`

Reply:
`gold`? This parameter is a list of the true labels so I figured it made sense?

Reply:
updated